javaparser-testing/src/test/java/com/github/javaparser/printer/PrettyPrintVisitorTest.java

```diff
@@ -130,4 +130,13 @@ public void printLambdaIntersectionTypeReturn() {
 
         assertEquals("return (Comparator<Map.Entry<K, V>> & Serializable) (c1, c2) -> c1.getKey().compareTo(c2.getKey());", print(methodDeclaration.getBody().get().getStatements().get(0)));
     }
+
+    @Test
+    public void printClassWithoutJavaDocButWithComment() {
+        String code = "/** javadoc */ public class A { \n// stuff\n}";
+        CompilationUnit cu = JavaParser.parse(code);
+        PrettyPrinterConfiguration ignoreJavaDoc = new PrettyPrinterConfiguration().setPrintJavaDoc(false);
+        String content = cu.toString(ignoreJavaDoc);
+        assertEquals("public class A {\n    // stuff\n}\n", content);
+    }
```

Comment:
Use the Utils.EOL constant instead of \n so the test doesn't fail on Windows :-(